manifests/migration.pp

```diff
@@ -5,37 +5,37 @@
 ) {
 
   Exec {
-    path => ['/sbin', '/bin', '/usr/sbin', '/usr/bin']
+    path => ['/sbin', '/bin', '/usr/sbin', '/usr/bin'],
   }
 
   exec { "stop ${service} for drbd migration":
     command => "service ${service} stop",
     # this should probably be more precise. If we're not linked into the
     # drbd volume we still would need to migrate
     unless  => "test -L ${name}",
-    require => Drbd::Resource[$volume]
+    require => Drbd::Resource[$volume],
   }
 
   if $ha_primary {
     exec { "migrate ${service} to drbd":
       command     => "mv ${name} /drbd/${volume}/${service}",
       subscribe   => Exec["stop ${service} for drbd migration"],
       before      => File[$name],
-      refreshonly => true
+      refreshonly => true,
```

Comment:
according to @roidelapluie, we should rewrite this whole thing into a type/provider because it has a refreshonly
i agree
this could be *very* dangerous if there's some kind of hiccup.